addons/pos_restaurant/static/src/js/Screens/ProductScreen/ControlButtons/PrintBillButton.js

```diff
@@ -13,6 +13,7 @@ odoo.define('pos_restaurant.PrintBillButton', function(require) {
         }
         async onClick() {
             const order = this.env.pos.get_order();
+            order.initialize_validation_date();
             if (order.get_orderlines().length > 0) {
                 await this.showTempScreen('BillScreen');
```

Comment:
```suggestion
            if (order.get_orderlines().length > 0) {
                order.initialize_validation_date();
                await this.showTempScreen('BillScreen');
```
No point in initializing a date in a case where it's not used

Reply:
> 

Thank you I pushed the new code